Onboarding.md

```diff
@@ -0,0 +1,22 @@
+# Onboarding/Offboarding
+
+Please refer to the text on Membership in the [TC Charter](TC-Charter.md).
+In the event that there is a contradiction between that document
+and this document, the contradictory elements in this document are
+invalid and are superseded by the TC Charter.
+
+## Onboarding
+
+* Confirm that the new member has read, understands, and agrees to uphold the
+  [Code of Conduct](Code-Of-Conduct.md).
+<!-- * Add the new member to the `@USGS-Astrogeology/ISIS_TC` team. -->
```

Comment:
So I tried to make an ISIS_TC team so it's easier to request reviews and such, but it requires that everyone be a member of the usgs-astrogeology org. I don't see a problem with that, but it's a bit odd.

Reply:
Yeah, I wasn't sure about those elements, so I commented them out until we get a handle on them.